changelog.md

```diff
@@ -1,4 +1,5 @@
 # Future
+- [FIXED] Properly pass options to sequelize.query in removeColumn [MSSQL] [#7139](https://github.com/sequelize/sequelize/pull/7193)
```

Comment:
@sushantdhiman Sorry about missing those. Thanks for catching. I've pushed with the suggested changes.